<launch>
    <!-- Set parameters -->
    <param name="look_ahead_dist" value="0.068"/>
    <param name="wheel_base" value="0.034"/>
    <param name="desired_speed" value="20.0"/>

    <!-- Launch the Python node -->
    <node pkg="control"
          type="control.py"
          name="control"
          output="screen">
    </node>

</launch>
